kolibri/core/assets/src/views/KSelect/KeenUiSelect.vue

```diff
@@ -211,13 +214,22 @@
           return [];
         },
       },
-      placeholder: String,
-      icon: String,
+      placeholder: {
+        type: String,
+        default: null,
```

Comment:
```suggestion
        default: '',
```

Since this might actually put a string literal `"null"` in some situations.